package.json

```diff
@@ -23,6 +23,7 @@
   "license": "BSD-3-Clause",
```

Comment:
Can we update the author and the license? and maybe set the version to 0.2.0... I am thinking about increasing the second number in every milestone and leave the last one for minor hotfixes in between sprints.